Firmware/MotorControl/motor.hpp

```diff
@@ -50,6 +51,9 @@ class Motor {
         float overcurrent_trip_level; // [A]
     };
 
+    float cached_v_current_control_integral_d_;
+    float cached_v_current_control_integral_q_;  
```

Comment:
Why do these values need to be cached? It is not faster to access them here than under the `current_control_` struct.

Reply:
The values are best recorded at the very end of the steady-state part of the control loop, just before the motor goes into its freewheel state. If measured after the freewheel state then it is not guaranteed the measurements were not made under transient conditions.

Reply:
I see, thanks for the explanation. I suggest removing "cached" from the name, since it is not possible to recalculate it on demand. It is more of a latched value then.

Reply:
I renamed it saved_steady_state_x which seemed more descriptive of its use.